actionpack/test/controller/caching_test.rb

```diff
@@ -53,12 +51,17 @@ def setup
     @controller.params = @params
     @controller.request = @request
     @controller.response = @response
+
+    @m1v1 = ModelWithKeyAndVersion.new("model/1", "1")
+    @m1v2 = ModelWithKeyAndVersion.new("model/1", "2")
+    @m2v1 = ModelWithKeyAndVersion.new("model/2", "1")
+    @m2v2 = ModelWithKeyAndVersion.new("model/2", "2")
   end
 
   def test_fragment_cache_key
-    assert_equal "views/what a key", @controller.fragment_cache_key("what a key")
```

Comment:
Maybe keep the tests for `fragment_cache_key` and add new ones for `combined_fragment_cache_key`? is deprecated we need to keep its tests otherwise the behavior can change and we don't notice.